proto/internalapi/central/compliance_operator.proto

```diff
@@ -57,9 +57,8 @@ message ApplyComplianceScanConfigRequest {
 
     string                  id              = 1;
     oneof scan_request {
-        OneTimeScan         one_time_scan   = 2;
-        ScheduledScan       scheduled_scan  = 3;
-        RerunScheduledScan  rerun_scan      = 4;
+        ScheduledScan       scheduled_scan  = 2;
+        RerunScheduledScan  rerun_scan      = 3;
```

Comment:
I worry about backwards compatibability, as we now change the ID of the field here (the case being a newer central sends a `msToSensor` with the updated field IDs and a old sensor tried to unmarshal the bytes and gets confused).

I'd prefer if we just say `reserved 2` and keep the existing messages as-is w.r.t their field ID.

Reply:
thanks for pointing it out, I have create a new PR in https://github.com/stackrox/stackrox/pull/9312